docs/docs/spark-getting-started.md

```diff
@@ -188,6 +188,12 @@ This type conversion table describes how Iceberg types are converted to the Spar
 | struct                     | struct                  |               |
 | list                       | array                   |               |
 | map                        | map                     |               |
+| nanosecond timestamp       |                         | Not supported |
+| nanosecond timestamp with timezone |                 | Not supported |
+| unknown                    |                         | Not supported |
+| variant                    |                         | Not supported |
```

Comment:
There's a bit of nuance here depending on the spark version. In Spark 4, Iceberg variant is mapped to Spark VariantType. This would be in the 1.10 release.